src/App.tsx

```diff
@@ -15,71 +18,84 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handlerAddGood = (good: string) => this.setState({ selectedGood: good });
```

Comment:
```suggestion
  handleAddGood = (good: string) => this.setState({ selectedGood: good });
```

Reply:
But why cant you do just `this.setState({selectedGood:good})` what is the purpose of this function

Reply:
it was corrected